pypeit/spectrographs/mmt_mmirs.py

```diff
@@ -0,0 +1,361 @@
+"""
+Module for MMT MMIRS
+"""
+import glob
+import numpy as np
+from scipy.signal import savgol_filter
+from astropy.time import Time
+from astropy.io import fits
+from astropy.stats import sigma_clipped_stats
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import parse
+from pypeit.core import framematch
+from pypeit.par import pypeitpar
+from pypeit.images import detector_container
+from pypeit.spectrographs import spectrograph
+from pkg_resources import resource_filename
+
+
+class MMTMMIRSSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle MMT/MMIRS specific code
+
+
+    """
+    ndet = 1
+
+    def __init__(self):
+        # Get it started
+        super(MMTMMIRSSpectrograph, self).__init__()
+        self.spectrograph = 'mmt_mmirs'
+        self.telescope = telescopes.MMTTelescopePar()
+        self.camera = 'MMIRS'
+
+    @staticmethod
+    def default_pypeit_par():
+        """
+        Set default parameters for the reductions.
+        """
+        par = pypeitpar.PypeItPar()
+        return par
```

Comment:
Do you need this method?  I think it's overwritten below.  The one below should be a `staticmethod`.

Reply:
Removed.